packages/twenty-front/src/pages/object-record/__stories__/RecordIndexPage.stories.tsx

```diff
@@ -43,10 +44,31 @@ export const Default: Story = {
   },
 };
 
-export const Loading: Story = {
+export const PrefetchLoading: Story = {
```

Comment:
I feel both stories (Prefetch and UserMetadata) should not be part of RecordIndexPage.stories.tsx, it's app level concepts. Can we add them under App instead?

Reply:
Of course!